tests/PushServiceTest.php

```diff
@@ -44,7 +44,7 @@ public static function setUpBeforeClass()
      */
     protected function setUp()
     {
-        if (!(getenv('TRAVIS') || getenv('CI'))) {
+        if (getenv('TRAVIS') || getenv('CI')) {
```

Comment:
Uh? No please don't skip the integration tests :o

Reply:
This one threw me off, maybe I'm misunderstanding though; the call to `$this->markTestSkipped` specifies "This test does not run on Travis". The conditional logic here shows that unless both `getenv('TRAVIS')` as well as `getenv('CI')` return false, skip the test. So I think, unless I'm just confusing myself at this point, I've just simplified the existing logic to the same effect? 🤔